publishing-api/config.ru

```diff
@@ -0,0 +1,14 @@
+require 'sidekiq'
+
+Sidekiq.configure_client do |config|
+  config.redis = {
+    host: ENV["REDIS_HOST"] || "127.0.0.1",
+    port: ENV["REDIS_PORT"] || 6379,
+    namespace: "publishing_api",
+  }
+end
+
+require 'sidekiq/web'
+map '/publishing-api' do
+  run Sidekiq::Web
+end
```

Comment:
In this file, we have `publishing_api` and `publishing-api` – would it make sense to use the same thing everywhere?